controllers/discordactions.js

```diff
@@ -121,7 +120,41 @@ const addGroupRoleToMember = async (req, res) => {
     return res.boom.badImplementation(INTERNAL_SERVER_ERROR);
   }
 };
+/**
+ * Patch Update user nickname
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const changeNicknameOfUsers = async (req, res) => {
```

Comment:
shouldn't the business logic be inside controller

Reply:
same reason @heyrandhir https://github.com/Real-Dev-Squad/website-backend/pull/1169#discussion_r1237127130